reminder.plugin.zsh

```diff
@@ -1,10 +1,14 @@
 TODO_SAVE_TASKS_FILE="$HOME/.todo.sav"
 TODO_SAVE_COLOR_FILE="$HOME/.todo_color.sav"
 
+# Using '#' because its pretty uncommon to type
+# typeset arrays by default use ':' but that means we can't have iTerm2 clickable URLs!
+TODO_SEP=\#
```

Comment:
Maybe you can use a construct like: 
```suggestion
TODO_SEP="${TODO_SEP:-"\#"}"
```
This would allow to setup the separator without modifying this file, just by setting this variable before loading the plugin.

Reply:
Nice, I'm new to shell scripting and was wondering how to do that.